dali/dfu/dfurun.cpp

```diff
@@ -593,12 +593,63 @@ class CDFUengine: public CInterface, implements IDFUengine
         unsigned auditflags = (DALI_LDAP_AUDIT_REPORT|DALI_LDAP_READ_WANTED);
         if (write)
             auditflags |= DALI_LDAP_WRITE_WANTED;
+
+        StringBuffer traceName;
+        fd->getTraceName(traceName);
+        if (traceName.length() > 255)
+        {
+            traceName.setLength(255);
+            traceName.append("...");
+        }
```

Comment:
This block is not needed unless access is denied, should be moved inside conditional.